source/defender.c

```diff
@@ -1,5 +1,5 @@
 /*
- * AWS IoT Device Defender Client v1.1.0
+ * AWS IoT Device Defender Client v1.1.1
  * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
Do we want to update 2020 to 2021?

Reply:
We havent updated them in other libraries. Should we?